.drone.yml

```diff
@@ -32,31 +32,56 @@ steps:
 kind: pipeline
 name: deploy to demo
 
+concurrency:
+  limit: 1
+
+depends_on:
+- qemu
+
 steps:
 - name: do it
-  image: docker
+  image: python:3.7-stretch
   privileged: true
   volumes:
   - name: docker-sock
     path: /var/run/docker.sock
+  - name: node
+    path: /opt/node
+  - name: cluster-var
+    path: /opt/cluster/var
   commands:
-  - while docker ps deploy-demo; do sleep 5 && echo waiting; done
-  - CONTAINER=$(./vagrant/launch-deploy-demo-container)
-  - docker logs -f $CONTAINER &
-  - sleep 10
-
-depends_on:
-- qemu
+  - cd /opt/node
+  - curl https://get.docker.com/ | sh -
+  - apt-get -yqq update
+  - apt-get -yqq install python3-venv python3-pip git curl unzip
+  - pip3 install pipenv
+  - pipenv install
+  - git --version
+  - docker --version
+  - python --version
+  - ./liquid alloc drone drone
+  # - liquid liquid.py halt
+  # - sleep 13
+  - git fetch -ap
+  - git reset --hard origin/demo
```

Comment:
I'd prefer if this step failed when there is a dirty working tree. Call me paranoid or old-fashioned if you must. :)

https://remarkablemark.org/blog/2017/10/12/check-git-dirty/